integration/env_test.go

```diff
@@ -146,35 +146,26 @@ func (e env) Exec(msg string, steps step.Steps, options ...execOption) (ok bool)
 }
 
 const (
-	Launchpad = "launchpad"
-	Stargate  = "stargate"
+	Stargate = "stargate"
```

Comment:
I think, I prefer to not complately remove these version namespaces and plugin system from the source code in case we need to use them to support multiple versions of the sdk again in the future.